### ISSUE TITLE
[Bug]: CN运行过程中Crash

### ISSUE THREAD

**Issue by xiaoshuwei** [matrixorigin/matrixone#12568]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): d17a04928bd4486a3772eeb7bd55781b8c4ef3ce
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

执行过程中突然出现CN Crash，导致SQL执行失败。

### Expected Behavior

保持正常执行。

### Steps to Reproduce

```Markdown
偶发
```

### Additional information

日志文件如下：
[cn-logs.txt](https://github.com/matrixorigin/matrixone/files/13291707/cn-logs.txt)

**Comment by volgariver6** [comment]
从这个日志看是没有磁盘空间了啊

```
panic: write /var/lib/matrixone/data/.tmp/3622586250.tmp: no space left on device
```

**Comment by sukki37** [comment]
closed as tracking by #12578
